pkg/auth/identity/ark_identity.go

```diff
@@ -435,7 +431,16 @@ func (ai *ArkIdentity) pickMechanism(challenge *identity.Challenge) (*identity.M
 	prompt := &survey.Select{
 		Message: "Please pick one of the following MFA methods",
 		Options: options,
-		Default: defaultChoice,
+	}
+
+	if ai.mfaType != "" {
```

Comment:
Default choice is not taken into account anymore i suppose? also, please bump VERSION and pyporject.toml

also, please look at the master build and align golint fixes (specifically the parsedUrl from the last PR